.gitignore

```diff
@@ -1,3 +1,5 @@
 .DS_Store
 node_modules
```

Comment:
storeing node_modules in git bad idea and bad practice

```sh
git clone yours_git_prj.git
cd yours_git_prj.git
yarn # or npmc i or npm i
# then
npm run dev
```

`examples/node_modules` for what? `budo`?